app/modules/asset_groups/models.py

```diff
@@ -846,11 +834,12 @@ def get_asset_for_file(self, filename):
     def begin_ia_pipeline(self, metadata):
         assert len(metadata.detection_configs) == 1
         assert metadata.data_processed == CreateAssetGroupMetadata.DataProcessed.complete
+        import copy
 
         for sighting_meta in metadata.request['sightings']:
             new_sighting = AssetGroupSighting(
                 asset_group_guid=self.guid,
-                meta=dict(sighting_meta),
+                config=copy.deepcopy(sighting_meta),
```

Comment:
Just doing dict() only copies the first level down, sighting_meta is deleted when the calling method goes out of scope so doing deepcopy is safer